src/modules/edc-demo/components/contract-agreement-transfer-dialog/model/http-datasink-auth-header-type.ts

```diff
@@ -0,0 +1 @@
+export type HttpDatasinkAuthHeaderType = 'None' | 'Value' | 'Vault-Secret';
```

Comment:
Remove one of the two, rename the other to "HttpParamsFormAuthHeaderType" to be used for both HttpDatasource and HttpDatasink